front/screens/search/tabAroundMeInstruction.component.tsx

```diff
@@ -59,18 +59,17 @@ const TabAroundMeInstruction: FC<Props> = ({ articles }) => {
 
   const extractPoiTypesFromArticles = () => {
     const finalCartographieTypes: PoiType[] = [];
-    articles
-      .map((article) => article.cartographie_pois_types)
-      .map((types) =>
-        types?.map((type) => {
-          if (
-            !finalCartographieTypes
-              .map((finalType) => finalType.nom)
-              .includes(type.nom)
-          )
-            finalCartographieTypes.push(type);
-        })
+    articles.map((article) => {
```

Comment:
Un map sert à créer un tableau de la même longueur que le tableau d'origine.
Ici, la fonction itérative ne retourne rien et la valeur de retour du map n'est pas utilisée.

Si la boucle ne retourne rien : forEach.
Si elle retourne un tableau plus petit du même type : filter.
Si elle retourne un tableau de type différent : reduce.